libs/service/verifier_service.h

```diff
@@ -0,0 +1,18 @@
+/*
+ *  Copyright (c) Microsoft Corporation
+ *  SPDX-License-Identifier: MIT
+ */
+#pragma once
+
+#include "config.hpp"
+#undef VOID
+#include "platform.hpp"
+#define VOID void
```

Comment:
remove, VOID is not used below (and shouldn't be used by any cpp files that include this)